src/components/pagination/index.md.njk

```diff
@@ -0,0 +1,108 @@
+---
+title: Pagination
+description: Help users navigate collections of numbered pages like search results
+section: Components
+aliases:
+backlog_issue_id: 77
+layout: layout-pane.njk
+---
+
+{% from "_example.njk" import example %}
+
+Help users navigate forwards and backwards through a series of pages. For example, search results or guidance that's divided into multiple website pages — like [the GOV.UK mainstream guide format](https://govuk-prototype-kit.herokuapp.com/docs/templates/mainstream-guide).
+
+{{ example({group: "components", item: "pagination", example: "default", html: true, nunjucks: true }) }}
+
+## When to use this component
+
+Consider using pagination when:
+- showing all the content on a single page makes the page take too long to load
+- most users will only need the content on the first page or first few pages
+
+## When not to use this component
+
+Only break up content onto separate pages if it improves the performance or usability of your service.
+
+Avoid using the 'infinite scroll' technique to automatically load content when the user approaches the bottom of the page. This causes problems for keyboard users.
+
+Do not use this Pagination component for linear journeys — for example, where you’re asking the user to complete a form. Instead, use the [Button component](/components/button/) (usually a 'Continue' button) to let the user move to the next page — and a [Back link](/components/back-link/) to let them move to the previous page.
+
+## How it works
+
+Add the pagination component after the content on each page that you're paginating.
+
+[View an example of Pagination in a standard GOV.UK page template](in-page/index.html).
+
+Do not show pagination if there's only one page of content.
+
+Redirect users to the first page if they enter a URL of a page that no longer exists.
+
+## For smaller numbers of pages
+
+Use 'Previous' and 'Next' links to let users navigate through a small number of pages. Stack the links vertically, so they’re more obvious to screen magnifier users when they’re zoomed in:
+
+{{ example({group: "components", item: "pagination", example: "labels", html: true, nunjucks: true }) }}
+
+### Add link labels to describe pages 
+
+You can use link labels to give more context:
+
+{{ example({group: "components", item: "pagination", example: "labels-2", html: true, nunjucks: true }) }}
+
+## For larger numbers of pages
+
+Use a list-type layout if users need to navigate through large numbers of pages. For example, long lists of search results.
+
+{{ example({group: "components", item: "pagination", example: "default", html: true, nunjucks: true, titleSuffix: "second" }) }}
```

Comment:
Reading the guide this bit feels slightly odd because the title says "larger numbers of pages" but the example only has three.

![Screenshot from 2022-06-12 14-40-45](https://user-images.githubusercontent.com/128088/173236067-838f9569-80b6-492e-81ef-fdc574488173.png)

None of the other examples on the page show ellipses, this might be a good place?

Reply:
This was a mistake in the updating of the guidance that the wrong example ended up on this section. We've updated it with the correct example.